operator/config/manifests/community/bases/loki-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1477 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Full Lifecycle
```

Comment:
I wondered a few times already if "Full Lifecycle" might be a bit too much for now. Should we lower this to the next lower one (which afaik is "Seamless Upgrades").

Reply:
Revisiting the [capability levels](https://sdk.operatorframework.io/docs/overview/operator-capabilities/#level-3---full-lifecycle) I still believe that the Loki Operator manages the full-lifecycle of loki. Couple of them don't really match to Loki (e.g. backup/restore), but we apply liveness/readiness probes, CPU/Memory requests, rolling restart strategy to Loki and the other operands. The only thing we miss is PodDisruptionBudget, but this is neglible for `1x.small` as it has only 2 replicas per component and might be only a thing to consider later for `1x.medium`'s ingesters and queriers.